src/components/Preferences/ModalActions.vue

```diff
@@ -47,4 +89,16 @@ export default Vue.extend({
     padding: var(--preferences-content-padding);
     border-top: 1px solid var(--header-border);
   }
+
+  .banner {
```

Comment:
I assume this class comes from `components/Banner.vue` because there's no such class in this file. There should be a comment to that effect.

Reply:
This is a good call out. I made this more explicit by adding a `banner-notify` class for us to apply these styles to.